morpheus/stages/preprocess/preprocess_nlp_stage.py

```diff
@@ -63,6 +63,8 @@ class PreprocessNLPStage(PreprocessBaseStage):
         overflowing token-ids can contain duplicated token-ids from the main sequence. If max_length is equal to stride
         there are no duplicated-id tokens. If stride is 80% of max_length, 20% of the first sequence will be repeated on
         the second sequence and so on until the entire sentence is encoded.
+    column : str
+        Name of the column containing the data that needs to be preprocessed.
```

Comment:
```suggestion
    column : str, default = "data"
        Name of the column containing the data that needs to be preprocessed.
```

Reply:
@mdemoret-nv I have updated the existing tests by adding a column argument to PreprocessNLPStage instantiation. Please let me know if I need to create a new function.

Reply:
This is more what I had in mind: https://github.com/nv-morpheus/Morpheus/pull/428/files#diff-b044cf5578e03f4186927f46fd55f286a8eea240249154287735b22c989f6b43R76